doc/dev/osd_internals/log_based_pg.rst

```diff
@@ -81,38 +81,27 @@ cannot.  It would be really annoying if we needed to have two entire
 implementations of PrimaryLogPG, one for each of the two, if there are
 really only a few fundamental differences:
 
-  1. How reads work -- async only, requires remote reads for ec
-  2. How writes work -- either restricted to append, or must
-     write aside and do a tpc
-  3. Whether we choose the oldest or newest possible head entry
-     during peering
-  4. A bit of extra information in the log entry to enable rollback
+1. How reads work -- async only, requires remote reads for ec
```

Comment:
done. I edit this on github's online editor. Need I squash these commits?

By the way. The ceph docuemnt website is broken. I sent a email to dev-maillist yesterday. It's still broken now.